src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamSystemDefaultsTest.cs

```diff
@@ -79,7 +79,7 @@ public async Task ClientAndServer_OneOrBothUseDefault_Ok(SslProtocols? clientPro
             using (X509Certificate2 serverCertificate = Configuration.Certificates.GetServerCertificate())
             using (X509Certificate2 clientCertificate = Configuration.Certificates.GetClientCertificate())
             {
-                string serverHost = serverCertificate.GetNameInfo(X509NameType.SimpleName, false);
+                string serverHost = Guid.NewGuid().ToString("N") + "." + serverCertificate.GetNameInfo(X509NameType.SimpleName, false);
```

Comment:
since we have GUID, the name will not match. I'm not sure if appending the name from certificate have any value here and it seems unnecessary. 
_if_ anything I would encode name of the test and parameters so it is easier too find particular connection in capture file - purely optional to make debug easier.

Reply:
You are right, the PR contains fix for both tests now.